CONTRIBUTING.md

```diff
@@ -309,7 +309,7 @@ All JavaScript must adhere to [JavaScript Standard Style](https://standardjs.com
 
 - Include thoughtfully-worded, well-structured [Jasmine](https://jasmine.github.io/) specs in the `./spec` folder.
 - Treat `describe` as a noun or situation.
-- Treat `it` as a statement about state or how an operation changes state.
+- Treat `it` as a statement about the state or how an operation changes state.
```

Comment:
The original is common parlance.

Reply:
Also this is inconsistent. If you want to talk about "the state", then it should be "the state" everywhere, not "the state" for the first instance and "state" for the next.